src/FluentValidation.AspNetCore/FluentValidationMvcExtensions.cs

```diff
@@ -116,16 +104,15 @@ public static class FluentValidationMvcExtensions {
 				}));
 			}
 
-			if (config.ClientsideEnabled) {

```

Comment:
Has the if statement here just been inverted? I can't see any functional changes, in which case please can this be reverted.